python/pyspark/mllib/regression.py

```diff
@@ -142,7 +149,8 @@ class LinearRegressionWithSGD(object):
 
     @classmethod
     def train(cls, data, iterations=100, step=1.0, miniBatchFraction=1.0,
-              initialWeights=None, regParam=0.0, regType=None, intercept=False):
+              initialWeights=None, regParam=0.0, regType=None, addIntercept=False,
```

Comment:
I'm sorry!  I got confused about "intercept," thinking it was being added to this class.  We should stick with the original name ("intercept") since it's a public API change otherwise.